model/group.cc

```diff
@@ -966,9 +973,9 @@ namespace minsky
   }
 
 
-  void Group::margins(float& left, float& right) const
+  void Group::margins(float& left, float& right, float& top, float& bottom) const
   {
-    left=right=10;
+    left=right=top=bottom=10;
```

Comment:
Top and bottom margins are constant, and don't need to be computed. Provided this doesn't require adding additional calls to margins(), which doesn't seem to have happened, this is OK.